pydis_site/templates/content/dropdown.html

```diff
@@ -0,0 +1,36 @@
+{% load str_methods %}
+
+<script>
+    document.addEventListener("DOMContentLoaded", () => {
+        const dropdown = document.querySelector("#dropdown");
+        // Show dropdown menu when clicked
+        dropdown.addEventListener("click", () => {
+            dropdown.classList.toggle("is-active");
+        });
+
+        // Hide dropdown menu when anyhere on the page is clicked
+        document.addEventListener("click", (e) => {
+            if (!dropdown.contains(e.target) && dropdown.classList.contains("is-active")) {
+                dropdown.classList.remove("is-active");
+            }
+        }, false);
+    });
+</script>
+
+<div class="dropdown is-pulled-right is-right" id="dropdown">
+    <div class="dropdown-trigger">
+        <a aria-haspopup="true" aria-controls="subarticle-menu">
+            <span>Sub-Articles</span>
+            <span class="icon is-small">
+                <i class="fas fa-angle-down" aria-hidden="true"></i>
+            </span>
+        </a>
+    </div>
+    <div class="dropdown-menu" id="subarticle-menu" role="menu">
+        <div class="dropdown-content">
+            {% for page in subarticles|dictsort:0 %}
+                <a href="{{page}}" class="dropdown-item">{{ page.title|replace_hyphens:" " }}</a>
```

Comment:
While this is a simple and fast approach, it is too naïve and capitalises all words. Furthermore, file names may not necessarily be anything close to the titles in the metadata (unless we start saying that they should be).

Reply:
Hmm, I didn't think of that. I like the filename freedom, as we want the site URL to be as concise as possible, while having a more descriptive page title.